roles/profiles/logging/meta/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+dependencies:
+  - elasticsearch
+  - kibana
```

Comment:
kibana/proxy should be there instead, so that it installs also httpd proxy above kibana server